sdcm/utils/nemesis_utils/node_operations.py

```diff
@@ -0,0 +1,64 @@
+import contextlib
+import logging
+
+from typing import Optional
+from random import choice
+
+from sdcm.cluster import BaseNode
+
+LOGGER = logging.getLogger(__name__)
+
+
+@contextlib.contextmanager
```

Comment:
This func definitely will not work on K8S.
Can you please add check for K8S and raise explicit exception, i.e. `NotImplementedError`?

Reply:
Disable nemesis for K8S